### ISSUE TITLE
@builder/sdk-react NextJS hydration warning

### ISSUE THREAD

**Issue by sfcgeorge** [BuilderIO/builder#2533]
**Describe the bug**

When using `@builder/sdk-react` with NextJS pages directory via getStaticProps there is a hydration warning in the console:

```
client.js:1 Warning: Prop `dangerouslySetInnerHTML` did not match. Server
```

**To Reproduce**

https://stackblitz.com/edit/stackblitz-starters-sfowpc?file=pages%2F%5B%5B...index%5D%5D.jsx

**Expected behavior**

I don't actually know if this matters but I assume we should pay attention to it.

**Screenshots**

<img width="1540" alt="Screenshot 2023-08-04 at 17 20 36" src="https://github.com/BuilderIO/builder/assets/885003/c892f596-c473-4b0a-9282-8d8882bfc90b">

**Comment by dansteele** [comment]
Hey @samijaber I've revisited this to see if it was fixed, but we're still blocked from properly moving to builder in our stack. Are there plans to work on this, or is there anything else we can do to help?

**Comment by samijaber** [comment]
We have published a fix for this in https://github.com/BuilderIO/builder/releases/tag/%40builder.io%2Fsdk-react%400.7.4

Let me know if you find any other weirdness 🙏🏽

**Comment by sfcgeorge** [comment]
Thanks, I can't seem to get it working on Stackblitz though; getting an `isolated_vm` error on install? I've seen that before in relation to Builder but never figured out how to fix it, just kinda changed things at random until it went away. https://stackblitz.com/edit/stackblitz-starters-6m6zgy

**Comment by samijaber** [comment]
@sfcgeorge Thank you for sharing that. That's a separate issue related to CJS/ESM bundling. Appreciate you pointing it out, will release a fix shortly and let you know.

**Comment by samijaber** [comment]
This is blocked by NextJS bug: https://github.com/vercel/next.js/issues/60491

**Comment by samijaber** [comment]
@sfcgeorge new release 0.8.1 addresses these issues. Unfortunately, Stackblitz acts up regarding `isolated-vm`. Not sure what the fix is for there right now, but your same code should work with 0.8.1 on a local machine
